.vscode/launch.json

```diff
@@ -4,6 +4,19 @@
 	// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
 	"version": "0.2.0",
 	"configurations": [
+		{
+			"type": "node",
+			"request": "launch",
+			"name": "flub buildVersion Mocha Tests",
+			"cwd": "${workspaceFolder}/build-tools/packages/build-cli",
+      "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/mocha",
+			"windows": {
+				"runtimeExecutable": "${workspaceFolder}/node_modules/.bin/mocha.cmd",
+			},
+			// "program": "${workspaceFolder}/common/tools/node_modules/mocha/bin/_mocha",
+			"args": ["--forbid-only", "test/**/*.test.ts", "--grep 'beta version'"],
+			"internalConsoleOptions": "openOnSessionStart",
+		},
 		{
```

Comment:
Should this be the first in the list?
I see commented out line too - did you mean to include this?